docker-config/fakes3/Dockerfile

```diff
@@ -1,4 +0,0 @@
-FROM lphoward/fake-s3:latest
```

Comment:
We should keep this as a reference to the image we're using in the dev deployment.